packages/target-electron/runtime-electron/runtime.ts

```diff
@@ -319,8 +319,12 @@ class ElectronRuntime implements Runtime {
     }
   }
   transformStickerURL(sticker_path: string): string {
-    return sticker_path
+    return encodeURI(`file://${sticker_path}`).replace(
+      /[?#]/g,
+      encodeURIComponent
+    )
```

Comment:
what is this replace for?

Reply:
It encodes ? and # if they are part of a folder name. I tested it, if they are neccessary. For a folder named various%20animals%201.1?id=3#45 they are neccessary

Reply:
ok